world/boards/templatetags/boards_tags.py

```diff
@@ -14,3 +14,7 @@ def render_post_summary(post):
 @register.simple_tag
 def get_board_detail_url(post):
     return reverse('boards:detail', kwargs={'post_id': post.id})
+
+@register.simple_tag
```

Comment:
Lines 18 to 20 seem to repeat lines 14 to 16. Is there supposed to be a get_board_detail_url and a get_post_detail_url?

Reply:
Uhhhh, past @ohboyotero was...out of line. Let's just pretend this didn't happen and I'll delete it.